src/Action/BaseAction.php

```diff
@@ -26,6 +26,15 @@
     protected $_responding = false;
 
     /**
+     * Default configuration
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'modelFactory' => 'Table',
```

Comment:
Every existing action declares `$_defaultConfig` so this is going to be overwritten. That's why all builds are failing.